test/Audit.OpenSearch.UnitTest/Audit.OpenSearch.UnitTest.csproj

```diff
@@ -0,0 +1,29 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFrameworks>net462;netcoreapp3.1</TargetFrameworks>
+		<DefineConstants>$(DefineConstants);STRONG_NAME</DefineConstants>
+		<NoWarn>$(NoWarn);1591;NETSDK1138</NoWarn>
+		<AssemblyName>Audit.Wcf.UnitTest</AssemblyName>
```

Comment:
Need to change this to the correct AssemblyName

Reply:
Done!